geoutils/georaster.py

```diff
@@ -1218,23 +1220,25 @@ def outside_image(self, xi,yj, index=True):
         :returns is_outside: True if ij is outside of the image.
         """
         if not index:
-            xi,xj = self.xy2ij(xi,yj)
+            xi,yj = self.xy2ij(xi,yj)
 
-        if np.any(np.array((xi,yj)) < 0):
+        if xi < 0 or yj < 0:
             return True
-        elif xi > self.ds.width or yj > self.ds.height:
+        elif xi > self.width or yj > self.height:
             return True
         else:
             return False
 
-    def interp_points(self,pts,nsize=1,mode='linear',band=1):
+    def interp_points(self,pts,input_latlon=False,nsize=1,mode='linear',band=1, **kwargs):
 
         """
         Interpolate raster values at a given point, or sets of points.
 
        :param pts: Point(s) at which to interpolate raster value. If points fall outside of image,
        value returned is nan.'
        :type pts: array-like
```

Comment:
What exactly is the expected format? From l 1258 it seems like it is ( (x1, y1), (x2, y2)...)?
If so, mention that shape should be (N, 2).